FavoritesMicro/Sources/FavoritesMicro/Favorites/FavoritesViewModel.swift

```diff
@@ -1,3 +1,30 @@
 import Foundation
+import SwiftUI
+import ApiService
+import UserData
+import Combine
 
-public class FavoritesViewModel: ObservableObject {}
+public class FavoritesViewModel: ObservableObject {
+    
+    @Published var favoriteMovies = UserData.shared
```

Comment:
this variable is referring to the UserData, and the name is confusing. rename this 
```suggestion
    @Published var userData = UserData.shared
```